packages/react-slider/src/components/RangedSlider/RangedSlider.test.tsx

```diff
@@ -0,0 +1,562 @@
+      expect(lowerInputRef.current?.min).toEqual('11');
+      expect(upperInputRef.current?.min).toEqual('11');
+    });
+
+    it('applies the max prop', () => {
+      const lowerInputRef = React.createRef<HTMLInputElement>();
+      const upperInputRef = React.createRef<HTMLInputElement>();
+      render(<RangedSlider max={11} inputLower={{ ref: lowerInputRef }} inputUpper={{ ref: upperInputRef }} />);
+      expect(lowerInputRef.current?.max).toEqual('11');
+      expect(upperInputRef.current?.max).toEqual('11');
+    });
+
+    it('applies the step prop', () => {
+      const lowerInputRef = React.createRef<HTMLInputElement>();
+      const upperInputRef = React.createRef<HTMLInputElement>();
+      render(<RangedSlider step={11} inputLower={{ ref: lowerInputRef }} inputUpper={{ ref: upperInputRef }} />);
+      expect(lowerInputRef.current?.step).toEqual('11');
+      expect(upperInputRef.current?.step).toEqual('11');
+    });
+
+    it('clamps an initial defaultValue that is out of bounds', () => {
+      const lowerInputRef = React.createRef<HTMLInputElement>();
+      const upperInputRef = React.createRef<HTMLInputElement>();
+      render(
+        <RangedSlider
+          defaultValue={[-10, 110]}
+          inputLower={{ ref: lowerInputRef }}
+          inputUpper={{ ref: upperInputRef }}
+        />,
+      );
+      expect(lowerInputRef.current?.value).toEqual('0');
+      expect(upperInputRef.current?.value).toEqual('100');
+    });
+  });
+
+  it('sorts an unsorted set of defaultValues', () => {
+    const lowerInputRef = React.createRef<HTMLInputElement>();
+    const upperInputRef = React.createRef<HTMLInputElement>();
+    render(
+      <RangedSlider defaultValue={[100, 50]} inputLower={{ ref: lowerInputRef }} inputUpper={{ ref: upperInputRef }} />,
+    );
+    expect(lowerInputRef.current?.value).toEqual('50');
+    expect(upperInputRef.current?.value).toEqual('100');
+  });
+
+  it('handles role prop', () => {
+    render(<RangedSlider role="test" data-testid="test" />);
+    const sliderRoot = screen.getByTestId('test');
+    expect(sliderRoot.getAttribute('role')).toEqual('test');
+  });
+
+  it('applies ariaValueText', () => {
+    const lowerInputRef = React.createRef<HTMLInputElement>();
+    const upperInputRef = React.createRef<HTMLInputElement>();
+
+    const values = ['small', 'medium', 'large'];
+    const defaultValue: [number, number] = [1, 2];
+    const getTextValue = (value: number) => values[value];
+
+    render(
+      <RangedSlider
+        defaultValue={defaultValue}
+        ariaValueText={getTextValue}
+        inputLower={{ ref: lowerInputRef }}
+        inputUpper={{ ref: upperInputRef }}
+        min={1}
+        max={3}
+      />,
+    );
+
+    expect(lowerInputRef?.current?.getAttribute('aria-valuetext')).toEqual(values[defaultValue[0]]);
+    expect(upperInputRef?.current?.getAttribute('aria-valuetext')).toEqual(values[defaultValue[1]]);
+  });
```

Comment:
You might to put these under another `describe` block that deals with accessibility specifically.

Reply:
Good idea. I'll also add the accessibility tests for the `Slider` that were recently merged.